cupy/cuda/_environment.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+
+
+_cuda_path = ()
+_nvcc_path = ()
+
+
+def get_cuda_path():
```

Comment:
How about naming like as `get_cuda_path_with_cache`?

Reply:
I think caching is an internal behavior and it's not important for the caller.